README.md

```diff
@@ -28,16 +28,16 @@ Microsoft PowerToys is a set of utilities for power users to tune and streamline
 
 - Windows 11 or Windows 10 v2004 (19041) or newer.
```

Comment:
Can we use 20h1 too? And is this "year-based" version number directly synonym with build number? I think I remember reading once that these were not _that_ exact. But maybe I'm just nitpicking and I should go to bed...